lib/uber-s3/object.rb

```diff
@@ -89,6 +90,14 @@ def save
       if !meta.nil? && !meta.empty?
         meta.each {|k,v| headers["x-amz-meta-#{k}"] = v.to_s }
       end
+
+      # Tagging
+      if !tagging.nil? && !tagging.empty?
+        # Turn the tags hash into a query string
+        tagging_query_string = tagging.map { |k, v| "#{k}=#{v}" }.join('&')
```

Comment:
is there any chance that `k` or `v` will have characters that need to be escaped?